plugins/events-backend/migrations/20240523100528_init.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function up(knex) {
+  // The event bus only supports PostgresSQL
+  if (knex.client.config.client === 'pg') {
+    await knex.schema.createTable('event_bus_events', table => {
+      table.comment('Events published to the events bus');
+      table
+        .bigIncrements('id')
+        .primary()
+        .comment('The unique ID of this event');
+      table
+        .dateTime('created_at')
+        .defaultTo(knex.fn.now())
+        .notNullable()
+        .comment('The time that the event was created');
+      table.text('topic').notNullable().comment('The topic of the event');
```

Comment:
no indexing on topics?

Reply:
Added, it didn't do anything to start, but after rewriting the query a bit it was actually used properly, more on that below.